modules/getting-started/con-audit-log-file-rotation-overview.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+// assembly-audit-log.adoc
+//
+
+[id="con-audit-log-file-rotation-overview_{context}"]
+= About audit log file rotation
```

Comment:
I think this heading is user-friendly, and incorporating product names can also improve SEO. wdyt?
```suggestion
= Audit log file rotation in {product}
```